File: Desafios em C#/1074 - Par ou Ímpar.cs
using System; 

//Developed by: @LucasMarcuzo
class URI1074 {
    static void Main(string[] args) { 

        int N,X,a;
    
        N=int.Parse(Console.ReadLine());

        for(a=1;a<=N;a++){

            X=int.Parse(Console.ReadLine());

            if(X==0){

                Console.WriteLine("NULL");

            }else if(X<=0&&X%2==0){

                Console.WriteLine("EVEN NEGATIVE");

            }else if(X<=0&&X%2==-1){

                Console.WriteLine("ODD NEGATIVE");

            }else if(X>=0&&X%2==0){

                Console.WriteLine("EVEN POSITIVE");

            }else if(X>=0&&X%2==1){

                Console.WriteLine("ODD POSITIVE");
            }
        }
    }
}
    

File: Desafios em JavaScript/1074 - Par ou Ímpar.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var N,X,a;
    
N=parseInt(lines.shift());

for(a=1;a<=N;a++){

    X=parseInt(lines.shift());

    if(X === 0){

        console.log("NULL");

    }else if(X<=0 && X % 2 === 0){

        console.log("EVEN NEGATIVE");

    }else if(X <= 0 && X % 2 === -1){

        console.log("ODD NEGATIVE");

    }else if(X >= 0 && X % 2 === 0){

        console.log("EVEN POSITIVE");

    }else if(X >= 0 && X %2 === 1){

        console.log("ODD POSITIVE");
    }
}